infra/modules/gcp-psoxy-rest/variables.tf

```diff
@@ -56,6 +56,11 @@ variable "deployment_bundle_object_name" {
   description = "Name of the object containing the deployment bundle"
 }
 
+variable "artifact_repository_id" {
+  type        = string
+  description = "ID of the artifact repository"
+}
```

Comment:
adding this w/o a default is a breaking change. older examples/etc that reference module directly (although shouldn't be common) will break unless we specify a default

Reply:
yeah - as above, it's a breaking change - can we put default as `null` or empty and deal with it?

Reply:
Yeah sorry, miss that before.